main/build.gradle

```diff
@@ -290,6 +293,9 @@ android {
 }
 
 dependencies {
+    // desugaring of Java 8 APIs
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.2.2'
```

Comment:
Version 1.2.2 is outdated meanwhile... Should be updated if we agree on it

Reply:
@fm-sys Please update the PR to the latest lib.
Then we can compare the apk size.

Reply:
done for testing purposes